### ISSUE TITLE
BLOCKED BY CONTROLLERS WITHOUT CHUNKED LOADING - Infinite scroll should display a loading icon / bobber when you reach the bottom + the endpoint hasn't yet responded.

### ISSUE THREAD

**Issue by jnaviask** [hicommonwealth/commonwealth#2877]
Let's make Notifications Paginate with Scroll so we have an example for other lists that need scrolling.

**Comment by CowMuon** [comment]
Closing as won't fix, we can reopen if needed for Unified Profiles

**Comment by masvelio** [comment]
What Gabe had in mind here is this https://virtuoso.dev/endless-scrolling/

https://user-images.githubusercontent.com/14819225/227206529-5317885e-6a44-4894-be8b-16a4fe51ae3f.mp4

To make it work, we have to have pagination correctly implemented on the backend side. It is very nice UX pattern but I don't think this is our biggest concern now. Thus, I would treat this ticket as low-priority or even won't fix. This should be agreed with the design that this is what we want, and the have a plan to introduce this to the every list which (and then this needs work on backend and then on the FE).

**Comment by masvelio** [comment]
needs investigation but it would need changes in:
- backend - to make api compatibile with endless scrolling pagination
- frontend - apply virtuoso & react-qeury for endless scrolling pagination
